packages/neon-core/src/sc/manifest/ContractFeatures.ts

```diff
@@ -0,0 +1,21 @@
+export interface ContractFeaturesLike {
+  storage: boolean;
+  payable: boolean;
+}
+
+export class ContractFeatures {
```

Comment:
ContractFeature.

Reply:
Features actually. 2 features now in this class

Reply:
do we really need a class for this? Looks like just a data object that we can put within the main class then. Is there going to be any operations we can do with it or are they just flags?

Reply:
Well I just want to use its `export` function when used in string <=> manifest conversion.
```json
"features": {
                "storage": true,
                "payable": true
            }
```

Reply:
Lets just simplify it and include it as part of the main class then.

Reply:
You mean to add a object-like prop in class `ContractManifest`? like below? 
```ts
features: {
  storage: boolean,
  payable: boolean
}
```

Reply:
Done